src/PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.cpp

```diff
@@ -16,6 +16,12 @@
 #include "Utilities/ErrorHandling/Error.hpp"
 #include "Utilities/MakeWithValue.hpp"
 
+#include "Parallel/Printf.hpp"
```

Comment:
I'd say remove the printf and defines. I'll make sure we have the defines globally with CMake :)